.github/workflows/tox.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-        - 3.8
+        - 3.9
```

Comment:
This should probably be submitted separately.

Reply:
Yep, I can do this switch before the rest, in one that also updates pip-compile version. It will make the history easier to read.